src/components/ebay-combobox/README.md

```diff
@@ -44,11 +44,15 @@ Name | Type | Stateful | Required | Description
 `borderless` | Boolean | No | No | whether the input has borders
 `disabled` | Boolean | Yes | No | sets the disabled attribute of the input
 `expanded` | Boolean | Yes | No | sets whether the listbox is expanded
-`autocomplete` | String | Yes | No | default is `none`; available values are `none` and `list`
+`autocomplete` | String | Yes | No | default is `none`; available values are `none`,  `list`, and `automatic`. See below for more info.
```

Comment:
So it's probably far from obvious, but this `autocomplete` attribute is intended to map to the `aria-autocomplete` attribute. Currently we only support `inline` and `none` from ARIA. We should move `automatic` to be controlled by a different attribute. Maybe call it `selection`?

More info here on "List autocomplete with manual selection" & "List autocomplete with automatic selection":

https://www.w3.org/TR/wai-aria-practices-1.1/#combobox

Reply:
Actually come to think of it, maybe we can still use the `autocomplete` attribute, but with the values as `list-manual` and `list-automatic`...? This would allow us to add `list-inline` in future (see WCAG example of that).

Reply:
Should the `roledescription` change for list automatic (like adding the input will be changed?) or fine leaving as is. Basically automatic and manual will be the same, options change based on text input:
```
        (`Use up and down arrow keys to navigate options.${
            input.autocomplete === 'none' ? '': ' Options change based on text input'
        }`);
```

Reply:
I think that's a good idea. Maybe, "Use up and down arrow keys to choose an option"? We can refine the text as we go. We need to keep it as short and sweet as possible.

I also notice a bug with the 1st default example. It shouldn't be announcing "options will change based on input".

Reply:
So the first one should be correct since it's `none`. 
I assume `none` should be the default behavior. Let me know if we should change the default?

Reply:
So sorry for the back and forth on this, but I think my instinct was right the first time, list selection type should be a separate attribute. This is because we also might want to configure a non-autocomplete combobox to be manual or automatic. So to summarize:

`autocomplete`: "none", "list" (in future we might also support ARIA values of "inline" and "both"
`list-selection`: "manual" or "automatic"

Where it might get a little quirky, is that for a basic combobox with `autocomplete` set to "none" (which are actually quite rare) we probably want `list-selection` to be manual by default. But for a combobox with `autocomplete` as "list", it probably makes more sense for `list-selection` to be automatic by default.

Reply:
Hopefully last one
I split up the `autocomplete` and `list-selection`.
So by default `list-selection` is `automatic` and only when they set it to `manual` then it doesn't select as they key up/down 
And they can mix `autocomplete` and `list-selection`

Reply:
Looks good. Can you please add an example with `list-selection` manual. We can cleanup and reorder these stories later as part of #1234

Reply:
There already is one, I moved some of the examples around: https://github.com/eBay/ebayui-core/pull/1247/files#diff-ccae6cf296486645387efb94ef728f77R1-R7

Reply:
Oh sorry, local env problem. I deleted example 03.